timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -1,13 +1,21 @@
 import logging
 import os
+import shutil
+from pathlib import Path
 from typing import Any, Dict, Literal, Optional, Union
 
 import numpy as np
 import pandas as pd
+from transformers import Trainer, TrainingArguments
```

Comment:
Ideally, we should hide the imports for all "large" libraries (like torch, transformers) inside class methods to avoid slow import times for `from autogluon.timeseries import ...`. We have to pay this overhead each time multiprocessing is used, so this adds up.

Reply:
Done.